src/_support/Page/Acceptance/Administrator/MenuItemFormPage.php

```diff
@@ -0,0 +1,150 @@
+<?php
+/**
+ * @package  Joomla.Test
+ * @subpackage  AcceptanceTester.Page
+ *
+ * @copyright   Copyright (C) 2005 - 2018 Open Source Matters, Inc.All rights reserved.
+ * @license     GNU General Public License version 2 or later;see LICENSE.txt
+ */
+namespace Page\Acceptance\Administrator;
+
+/**
+ * Acceptance Page object class to menuitem form page.
+ *
+ * @category  Users
+ * @package   Page\Acceptance\Administrator
+ * @copyright 2005 - 2018 Open Source Matters, Inc. All rights reserved.
+ * @license   GNU General Public License version 2 or later; see LICENSE.txt
+ * @since     4.0.0
+ */
+class MenuItemFormPage
+{
+	/**
+	 * Menu Item Title
+	 *
+	 * @var   string
+	 * @since 4.0.0
+	 */
+	public static $menuItemTitle = '#jform_title';
+
+	/**
+	 * Menu Item Alias
+	 *
+	 * @var   string
+	 * @since 4.0.0
+	 */
+	public static $menuItemAlias = '#jform_alias';
+
+	/**
+	 * Menu Item Save
+	 *
+	 * @var   string
+	 * @since 4.0.0
+	 */
+	public static  $saveButton = '#toolbar-apply';
+
+	/**
+	 * Page title of the user manager listing page.
+	 *
+	 * @var   string
+	 * @since 4.0.0
+	 */
+	public static $pageTitleText = "Menus";
+
+	/**
+	 * A drop down menu
+	 *
+	 * @var   string
+	 * @since 4.0.0
+	 */
+	public static $menuDropDown = '//select[@id="jform_menutype"]';
+
+	/**
+	 * Selecting option from dropdown menu
+	 *
+	 * @var   string
+	 * @since 4.0.0
+	 */
+	public static $selectOption = '//select[@id="jform_menutype"]/option[@value="mainmenu"]';
+
+	/**
+	 * Select Menu Type Modal
+	 *
+	 * @var    string
+	 * @since  4.0.0
+	 */
+	public static $menuTypeModal =  '#menuTypeModal';
+
+	/**
+	 * Select Article Link
+	 *
+	 * @var    string
+	 * @since  4.0.0
+	 */
+	public static $articlesLink = 'Articles';
+
+	/**
+	 * Archive Articles
+	 *
+	 * @var    string
+	 * @since  4.0.0
+	 */
+	public static $archiveArticles = 'Archived Articles';
+
+	/**
+	 * Success Message
+	 *
+	 * @var    string
+	 * @since  4.0.0
+	 */
+	public static $successMessage = 'Menu item saved';
```

Comment:
Same for this, searching for it gives me zero results ;)

Reply:
I used that for MenuItemCest file probably